ui/pages/swaps/build-quote/build-quote.test.js

```diff
@@ -175,7 +207,17 @@ describe('BuildQuote', () => {
     setFromTokenInputValue.mockImplementation(setFromTokenInputValueMock);
     const mockStore = createSwapsMockStore();
     mockStore.swaps.fromToken = 'DAI';
-    const store = configureMockStore(middleware)(mockStore);
+    const store = configureMockStore(middleware)({
+      ...mockStore,
+      metamask: {
+        ...mockStore.metamask,
+        ...mockNetworkState({
+          chainId: CHAIN_IDS.MAINNET,
+          blockExplorerUrl:
+            'https://etherscan.io/token/0xA0b86991c6218b36c1d19D4a2e9Eb0cE3606eB48',
```

Comment:
Same comment as https://github.com/MetaMask/metamask-extension/pull/26433/files#r1761786173

Reply:
fixed in https://github.com/MetaMask/metamask-extension/pull/26433/commits/97104e242472bd9263e59dbb9714271169ef0972